.env-template

```diff
@@ -9,7 +9,7 @@ MIGRATE_TOKEN=<MIGRATE_TOKEN>
 # TOOLS
```

Comment:
Is an env template needed now?

Reply:
Going to keep env's around also in the docs until this is tested more